### DISCUSSION TITLE
Welcome to EdgeDB Discussions!

### DISCUSSION THREAD

**Discussion by 1st1** [geldata/gel#1499] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by haikyuu** [comment]
Hi everyone. First of all, thanks for building this wonderful tool. ❤️

I am Abdellah Alaoui, a full stack developer from Morocco. My main tech stack is node and React. 

 I never liked a database technology before. Their learning curves are quite steep ... Until i discovered EdgeDB. I love its readable and powerful syntax and SDL.

I am currently building a new bootstrapped company with a cofounder using EdgeDB in the database layer. I think it will get to `1.0` by the time we launch.

Other than that, i do some freelancing and technical writing part time to pay the bills :)

**Comment by cthoyt** [comment]
Hi everyone! I’m a bioinformatician and I love working with graphs and ontologies. Also I hate foreign keys and love python, so EdgeDB seemed like a great fit for me. I’m playing around and trying to build an open source terminology management system on top of EdgeDB. I’ll keep you updated when there’s some progress :)

Thanks for building this nice tool

**Comment by dmgolembiowski** [comment]
Greetings, all! I'm a POS Systems Administrator at a company that makes heavy use of Oracle SQL databases.
In my personal time, I'm an avid learner of programming languages and I enjoy building everything from simple scripts to custom frameworks. EdgeDB caught my attention because it has a remarkable way to express data. If you haven't given EdgeDB's SDL, DDL, or EdgeQL a try, it's worth playing around with for a weekend to get your feet wet. Writing functions in EdgeQL feels fun, clever, and rewarding when you get it right. In the future, I'm hoping that EdgeDB will release a more robust module system: one that supports exporting local modules, uploading them with some command line interface to a central repository, and sharing them with other database or application developers.
